cmake/modules/SwiftComponents.cmake

```diff
@@ -97,10 +98,10 @@ macro(swift_configure_components)
   set(SWIFT_INSTALL_COMPONENTS "${_SWIFT_DEFAULT_COMPONENTS}" CACHE STRING
     "A semicolon-separated list of components to install from the set ${_SWIFT_DEFINED_COMPONENTS}")
 
-  # 'compiler' depends on 'swift-syntax-lib' component.
```

Comment:
I guess theoretically this could have been removed when we did the split 😅?

Reply:
Yeah, I totally forgot about this.